cmd/kube-controller-manager/app/core.go

```diff
@@ -126,6 +126,7 @@ func startNodeLifecycleController(ctx ControllerContext) (http.Handler, bool, er
 		ctx.InformerFactory.Core().V1().Pods(),
 		ctx.InformerFactory.Core().V1().Nodes(),
 		ctx.InformerFactory.Apps().V1().DaemonSets(),
+		// cloud node lifecycle controller uses existing cluster role from node-controller
```

Comment:
is this comment correct? is this the cloud node lifecycle controller?

Reply:
fixed, thanks!